README.md

```diff
@@ -12,11 +12,11 @@ If the app notices you that "the token or the project ID is not correct" please
 ## Quick Start Guide
 If it's your first time running this project, you should install all dependencies first:
 ```bash
-$ npm install
+$ npm install  OR  $ yarn install
```

Comment:
You don't need to write option `yarn` or `npm`, it's fine (even better) to mention just one. Since we're using `npm` I'd just mention that